x-pack/plugins/siem/public/components/formatted_date/index.tsx

```diff
@@ -6,26 +6,29 @@
 
 import moment from 'moment-timezone';
 import * as React from 'react';
+import { useContext } from 'react';
 import { pure } from 'recompose';
 
 import { isString } from 'lodash/fp';
-import { AppKibanaFrameworkAdapter } from '../../lib/adapters/framework/kibana_framework_adapter';
+import {
+  AppKibanaFrameworkAdapter,
+  KibanaConfigContext,
+} from '../../lib/adapters/framework/kibana_framework_adapter';
 import { getOrEmptyTagFromValue } from '../empty_value';
 import { LocalizedDateTooltip } from '../localized_date_tooltip';
 
-export const KibanaConfigContext = React.createContext<Partial<AppKibanaFrameworkAdapter>>({});
-
-export const PreferenceFormattedDate = pure<{ value: Date }>(({ value }) => (
-  <KibanaConfigContext.Consumer>
-    {(config: Partial<AppKibanaFrameworkAdapter>) => {
-      return config && config.dateFormat && config.dateFormatTz && config.timezone
+export const PreferenceFormattedDate = pure<{ value: Date }>(({ value }) => {
+  const config: Partial<AppKibanaFrameworkAdapter> = useContext(KibanaConfigContext);
+  return (
+    <>
+      {config && config.dateFormat && config.dateFormatTz && config.timezone
```

Comment:
When would `config` be undefined or null? It doesn't look like it ever would.

Reply:
Nice catch! I verified both the `KibanaConfigContext`'s default value, and the object it is set to (`AppFrontendLibs.framework`) can neither can be null. I'll go ahead and remove this additional check. Thanks!